docs/manual/manual.adoc

```diff
@@ -1619,6 +1619,7 @@ not considered local by the scanner:
 * `SOURCE_DATE_EPOCH` - Timestamp in seconds since epoch. This timestamp will be used instead of the current time to populate `timestamp` attributes in SCAP source data streams created by `oscap ds sds-compose` sub-module. This is used for reproducible builds of data streams.
 * `OSCAP_PROBE_MEMORY_USAGE_RATIO` - maximum memory usage ratio (used/total) for OpenSCAP probes, default: 0.1
 * `OSCAP_PROBE_MAX_COLLECTED_ITEMS` - maximal count of collected items by OpenSCAP probe for a single OVAL object evaluation
+* `OSCAP_PROBE_IGNORE_PATHS` - Skip given paths during evaluation. If multiple paths should be skipped they need to be separated by a colon. The paths should be absolute.
```

Comment:
So, I've checked some paths and found out that paths like `/a/b/../b/c` does not work (as well as `/a/./b/c`).

Which brings me to the conclusion that the requirement is actually to provide the *canonical* path, not one of absolute paths.